project/src/components/room/form/feedback-form/feedback-form.tsx

```diff
@@ -0,0 +1,40 @@
+import { FormEventHandler, useState } from 'react';
+import { ChangeEventHandlerCommon } from '../../../../types/handlers';
+import { Reviews } from '../../../../types/review';
+import FeedbackRating from '../feedback-rating/feedback-rating';
+
+type FeedbackFormProps = {
+  reviews: Reviews;
+}
+
+export default function FeedbackForm({reviews}: FeedbackFormProps) {
+  const [formData, setFromData] = useState({
+    userId: 0,
+    review: '',
+    rating: 0
+  });
+
+  const fieldChangeHandle: ChangeEventHandlerCommon = (evt) => {
+    const {name, value} = evt.target;
+    setFromData({...formData, [name]: value});
+  };
+
+  const submitFormHandle: FormEventHandler<HTMLFormElement> = (evt) => {
```

Comment:
Аналогично с неймингом + обычно используют form event
```suggestion
  const submitFormHandle: FormEvent<HTMLFormElement> = (evt) => {
```